### PR TITLE
Nxp 24314 disable wms from properrty

### PR DESCRIPTION
**Pull request by pierre-gautier**
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-pierregautier/233/

### CODE REVIEW

File: nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java [hunk 1]
```diff
@@ -189,7 +189,7 @@ public int getApplicationStartedOrder() {
     public void start(ComponentContext context) {
         super.start(context);
         ClusterService clusterService = Framework.getService(ClusterService.class);
-        if (clusterService.isEnabled()) {
+        if (clusterService != null && clusterService.isEnabled()) {
```
Comment (efge) [inline nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java]: Please use NXP-25499 for this fix.
But the fix should rather be to add the proper `@Deploy("org.nuxeo.runtime.cluster")` to the test that fails (until we have cleaned up all our features to have one feature per test service).

Comment (pierre-gautier) [reply nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java]: Ok for the NXP-25499
For the fix, I do not agree, it's a potential NPE if clusterService is not deployed tests or not, and apparently it's not a blocking case

Comment (efge) [reply nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java]: ClusterService is a basic runtime service that is required by pubsub and therefore for invalidations of caches, directories and repositories. It should always be deployed before the cache service.

Comment (efge) [reply nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheServiceImpl.java]: If we had proper dependencies between services we would make this explicit with injection or requirements...

**Comment by kevinleturc** [comment]
👍

File: nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java [hunk 2]
```diff
@@ -104,6 +104,11 @@
      */
     public static final String SHUTDOWN_DELAY_MS_KEY = "nuxeo.work.shutdown.delay.ms";
 
+    /**
+     * @since 11.1
+     */
+    public static final String WORKMANAGER_PROCESSING_DISABLE = "nuxeo.work.processing.disable";
```
Comment (funshodavid) [inline nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java]: `disabled` instead of `disable` ?

**Comment by pierre-gautier** [comment]
👍

**Pull request merged**